scripts/run.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+if [ $1 == "server" ]; then
+    exec /usr/local/bin/absearch-server /app/config/absearch.ini
+elif [ $1 == "nosetest" ]; then
+    # install dependencies (if required)
+
+    if [ $EUID != 0 ]; then
+        echo "Need to be root.  Run container with '--user root'"
+        exit 1
+    fi
+
+    apt-get install -y redis-server
+    pip install -r dev-requirements.txt
+    nosetests --nocapture absearch
+elif [ $1 == "flake8" ]; then
+    pip install flake8
+    flake8 absearch
```

Comment:
To be honest, it feels a little heavyweight to use Docker to run `flake8` on the source code. I'm OK with the use of Docker for the tests -- tests should conceptually be run against the "installed" version of the project, and I'm OK with that being a Docker container -- but `flake8` should be run against the repository, so it feels a little odd.